data/intuos-pro-2-s.tablet

```diff
@@ -0,0 +1,51 @@
+# Wacom
+# Intuos Pro 2 S
+# PTH-460
```

Comment:
This needs to (also) go into the new `ModelName` property, see b94cd3fcf97b3297943032e0f5b29e7cfbd29828